test/externalTests/euler.sh

```diff
@@ -66,6 +66,10 @@ function euler_test
     # TODO: Remove this when https://github.com/NomicFoundation/hardhat/issues/2453 gets fixed.
     sed -i "/expectError: 'JUNK_UPGRADE_TEST_FAILURE'/d" test/moduleUpgrade.js
 
+    # Workaround to make euler test pass
```

Comment:
I'd include a bit more info here. All of these patches are workarounds so it's good to have a quick way to discern them without having to visit the URL :)
```suggestion
    # One of the tests is currently broken in the upstream, due to changed behavior of blockid in @ethereumjs/vm.
```